arch/HPPA/HPPADisassembler.c

```diff
@@ -304,6 +302,18 @@ static void fillSysopInsnName(MCInst *MI, uint32_t insn) {
             return;
         case 0x65:
             push_str_modifier(&MI->hppa_ext, "r");
+            MCInst_setOpcode(MI, HPPA_INS_RFI);
+            if (MI->flat_insn->detail != NULL) {
+                cs_detail *detail = get_detail(MI);
+                detail->regs_write[detail->regs_write_count++] = HPPA_REG_GR1;
```

Comment:
Please check `Mapping.h`. There are functions defined for this `map_add_implicit_write` and others for implicit read/write.